lib/mp4/transmuxer.js

```diff
@@ -638,13 +691,37 @@ Transmuxer = function(options) {
         audioSegmentStream = new AudioSegmentStream(audioTrack);
 
         // Set up the final part of the audio pipeline
-        aacStream
+        adtsStream
           .pipe(audioSegmentStream)
           .pipe(coalesceStream);
       }
     }
   });
 
+  };
+
+  Transmuxer.prototype.init.call(this);
+  options = options || {};
+  aacfile = options.aacfile;
```

Comment:
It would only be used once if you followed my two recommendations in the comments following this one. Also it should definitely have a name like `aacInput`